.gitlab/kitchen_testing/new-e2e_testing/centos.yml

```diff
@@ -52,6 +52,22 @@ new-e2e-agent-platform-install-script-upgrade6-centos-x86_64:
   variables:
     FLAVOR: datadog-agent
 
+new-e2e-agent-platform-install-script-upgrade7-centos-x86_64:
+  stage: kitchen_testing
+  extends:
+    - .new_e2e_template
+    - .new-e2e_script_upgrade6
+    - .new-e2e_os_centos
+    - .new-e2e_centos_a6_x86_64
+    - .new-e2e_agent_a6
+  variables:
+    FLAVOR: datadog-agent
+    FROM_CURRENT_TO_STABLE: true
+  parallel:
+    matrix:
+      - START_MAJOR_VERSION: [6]
+        END_MAJOR_VERSION: [7]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is it necessary since there is only one combination?

Reply:
I agree I can get rid of the `matrix` directive, it was a simple copy/paste

Reply:
We cannot just use `variables`, because these jobs inherit from a template having a matrix. So I need to override that matrix, otherwise it will [fail like this](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/715526595) and have a bad name `new-e2e-agent-platform-install-script-upgrade7-suse-x86_64: [5, 6]`